README.md

```diff
@@ -29,14 +29,15 @@ any changes to the orb.
  - [ovotech/aws-rotate-keys@1](aws-rotate-keys) - Rotate AWS access keys and update corresponding CircleCI environment variables.
  - [ovotech/clair-scanner@1](clair-scanner) - Scan Docker images for vulnerabilities.
  - [ovotech/rac-gcp-deply@1](rac-gcp-deploy) - Deploy Scala services to a Kubernetes cluster running in Google Cloud.
- - [ovotech/set-current-environment@1](aws-get-parameters) - Set current environment (master branch, tag or other) to environemntal variable..
+ - [ovotech/set-current-environment@1](set-current-environment) - Set current environment (master branch, tag or other) to environmental variable.
```

Comment:
why are we changing that? Its not ours

Reply:
I think it was one of mine - anyway it's an error so I thought I'd fix it.

Reply:
ah fair enough